docs/readmes/lte/migration_guide.md

```diff
@@ -0,0 +1,38 @@
+---
+id: migration_guide
+title: AGW migration guide
+hide_title: true
+---
+
+# Access Gateway migration guide
+
+> NOTE: Only Ubuntu-based Access Gateways are supported starting from Magma release 1.6, Debian support is deprecated.
+
+For a smooth transition from a Debian to Ubuntu access gateway, it is advised to backup local configs including `RootCA`, `control_proxy.yml`, `Hardware ID` and `Challenge Key` files. It is also recommended to backup any static IPs or routes provisioned on the existing access gateway.
+
+### Before migration
+
+#### Copy the contents from the following file locations to a local backup storage:
+
+- `Control Proxy`
+> */var/opt/magma/configs/control_proxy.yml*
+
+- `Root CA`
+> */var/opt/magma/tmp/certs/rootCA.pem*
+
+- `Hardware ID`
+> */etc/snowflake*
+
+- `Challenge Key`
+> */var/opt/magma/certs/gw_challenge.key*
+
+After copying these, gather the output of `show_gateway_info.py` as reference and follow the [Ubuntu Install Guide](https://docs.magmacore.org/docs/lte/deploy_install_ubuntu)
```

Comment:
the deployment link is changed on master, can you change deploy_install_ubuntu  to deploy_install.